src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -527,7 +527,7 @@ private static ProcessInfo[] GetProcessInfos(PerformanceCounterLib library, int
                                 // at the end.  If instanceName ends in ".", ".e", or ".ex" we remove it.
                                 if (instanceName.Length == 15)
                                 {
-                                    if (instanceName.EndsWith(".", StringComparison.Ordinal))
+                                    if (instanceName.EndsWith("."))
```

Comment:
Is `StringComparison.CurrentCulture` better than `StringComparison.Ordinal` there?

Reply:
No, and this doesn't use CurrentCulture.

Reply:
Sorry, my bad. I thought it is default for `string.EndsWith(string)` and same for `ReadOnlySpan<char>` alternative.